web-app/components/pages/login/login.jsx

```diff
@@ -133,6 +184,16 @@ export const Login = () => {
                             <LoginButton variant="contained" size="medium" color="primary" type="submit" disableElevation>
                                 Login
                             </LoginButton>
+                            <Backdrop sx={(theme) => ({ color: '#fff', zIndex: theme.zIndex.drawer + 1 })} open={open} onClick={handleClose}>
```

Comment:
```suggestion
                            <Backdrop sx={(theme) => ({ color: '#fff', zIndex: theme.zIndex.drawer + 1 })} open={open}>
```
And remove `onClick`